clients/src/main/java/org/apache/kafka/clients/Metadata.java

```diff
@@ -200,7 +200,7 @@ public synchronized boolean containsTopic(String topic) {
      * Updates the cluster metadata. If topic expiry is enabled, expiry time
      * is set for topics if required and expired topics are removed from the metadata.
      */
-    public synchronized void update(Cluster cluster, long now) {
+    public synchronized void update(Cluster cluster, boolean hasUnavailablePartitions, long now) {
```

Comment:
I was wondering about that too. It seems consistent with the other things we have there.